src/courtroom.cpp

```diff
@@ -2604,8 +2604,16 @@ void Courtroom::append_ic_text(QString p_text, QString p_name, QString p_action,
 
   // Format the name of the actor
   ui_ic_chatlog->textCursor().insertText(p_name, bold);
+  // Special case for stopping the music
+  if (p_action == "has stopped the music") {
```

Comment:
```suggestion
  if (p_action == tr("has stopped the music")) {
```
Needs `tr()` - otherwise won't work as intended.